recipes/sole/all/conanfile.py

```diff
@@ -0,0 +1,39 @@
+from conans import ConanFile, tools, CMake
+import os
+
+
+class SoleConan(ConanFile):
+    name = "sole"
+    homepage = "https://github.com/r-lyeh-archived/sole"
+    description = "Sole is a lightweight C++11 library to generate universally unique identificators (UUID), both v1 and v4."
+    topics = ("conan", "uuid", "header-only")
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler"
+    no_copy_source = True
+    license = "zlib"
```

Comment:
```suggestion
    license = "Zlib"
```

https://spdx.org/licenses/Zlib